src/docs/product/data-management-settings/dynamic-sampling/index.mdx

```diff
@@ -43,6 +44,10 @@ While prioritizing data from the latest release, we also take into consideration
 
 We prioritize data from development environments because these environments typically generate a relatively small number of transactions compared to your production environments. As a result, to gain meaningful insights during your testing phase, we prioritize transaction data from these environments.
 
+### Key Transactions
+
+We prioritize my Key Transactions to 5x times from blended rate. This rule applies to single transactions.
```

Comment:
```suggestion
For most projects there are only a handful or transactions that you really care about and we purposefully designed the starring feature to give you a visual prioritization of them. With dynamic sampling, starred transactions are even more useful because we will prioritize them up to five times over the blended rate. This rule applies to single transactions.
```

Reply:
What do you think @imatwawana?

Reply:
thanks!